seewav.py

```diff
@@ -190,6 +212,22 @@ def visualize(audio,
     except (IOError, ValueError) as err:
         fatal(err)
         raise
+
+    output_size = size
+    image = None
+    if bg_image is not None:
+        try:
+            image = Image.open(bg_image)
+        except (IOError, ValueError) as err:
+            fatal(err)
+            raise
+        # resize image to be compatible with ffmpeg
+        if image.width % 2 is 1:
+            image = image.resize(image.width + 1, image.height)
+        if image.height % 2 is 1:
+            image = image.resize(image.width, image.height + 1)
+        output_size = image.width, image.height
```

Comment:
I'm not sure we should replace the output size with tjhe image size, maybe we should rather keep the video width and height controlled by the params, and resize the image given as background ? or maybe add a flag `--use_background_image_size`